lib/features/children/family_goal/pages/create_family_goal_page.dart

```diff
@@ -0,0 +1,70 @@
+import 'package:flutter/material.dart';
+import 'package:givt_app/core/enums/enums.dart';
+import 'package:givt_app/features/children/family_goal/widgets/family_goal_circle.dart';
+import 'package:givt_app/features/children/family_goal/widgets/family_goal_creation_stepper.dart';
+import 'package:givt_app/shared/widgets/custom_green_elevated_button.dart';
+import 'package:givt_app/utils/utils.dart';
+import 'package:go_router/go_router.dart';
+
+class CreateFamilyGoal extends StatelessWidget {
+  const CreateFamilyGoal({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        centerTitle: true,
+        title: Text(
+          //TODO: POEditor
+          'Create a Family Goal',
+          style: Theme.of(context).textTheme.titleLarge?.copyWith(
+                fontFamily: 'Mulish',
```

Comment:
Also we do not have the font 'Mulish' in this repo so it just defaults to basic sans serif (ugly). Either upload the mulish family ttfs manually to the fonts folder and pubspec or add it through google fonts